### DISCUSSION TITLE
Changing a http method

### DISCUSSION THREAD

**Discussion by shubhambajad** [apideck-libraries/portman#373] [Q&A]
Hi Team,

Actually I have one api request with GET http method but for negative testing I want to send it with POST request so that I should receive a response as 405. So is there any way to change the api http method?

**Comment by shubhambajad** [comment]
@thim81 Hi Can you also check this part whether we are able to do it or not I am also trying to do some solution for the same.

**Reply by thim81** [reply]
@shubhambajad Currently it is not supported to modify the HTTP methods.

I'll iterate on the idea with @nicklloyd , but I'm not sure about this feature.

**Reply by shubhambajad** [reply]
Hi @thim81 

So we can do it postman by setting the variable for method and we will pass the value of new http method from pre-requisite script.

So if we add one more overwriteProperty lets say overwriteHttpMethod it will help to check negative scenarios in case of invalid http method:

one link how we can do that in postman is: https://www.youtube.com/watch?v=PinpJPH6HSc

**Reply by shubhambajad** [reply]
@thim81 Can you check this one also ? whether any plan for this feature?

**Reply by shubhambajad** [reply]
Any updates on this? or any work around any one can suggest on this ? @thim81 @nicklloyd
